react-native-in-app-payments-quickstart/app/components/OrderTitleView.tsx

```diff
@@ -30,7 +30,7 @@ OrderTitleView.propTypes = {
   onCloseOrderScreen: PropTypes.func.isRequired,
 };
 
-export default function OrderTitleView({ onCloseOrderScreen }) {
+export default function OrderTitleView({ onCloseOrderScreen }: { onCloseOrderScreen: any }) {
```

Comment:
please use a concrete type instead of any

Reply:
Resolved it.